src/properties/output/index.test.js

```diff
@@ -39,7 +39,8 @@ const validModuleConfigs = [
   { crossOriginLoading: 'anonymous' },
   // #17
   { hashDigestLength: 6 }, // undocumented
-
+  // #18
+  { publicPath: '' },
```

Comment:
This shouldn't be in this PR. Can you clean up your commit history to only have relevant changes?